syntax/output_panel.sublime-syntax

```diff
@@ -22,3 +22,7 @@ contexts:
     - match: -> ([^ ]*)
       captures:
         1: storage
+
+    - match: \s(https?:\S+)
```

Comment:
Add Syntax test?

Reply:
Yeah, we don't have syntax tests for this syntax at all.  Need to create one from scratch.  https://github.com/timbrel/GitSavvy/issues/1514